.github/workflows/buildExamples.yml

```diff
@@ -36,3 +36,8 @@ jobs:
 
       - name: Build examples
         run: yarn workspaces foreach -p run build-storybook
+
+      - name: Publish examples to Chromatic
```

Comment:
I think this should be pulled out of the matrix and run in only one OS.  Otherwise we're going to try to publish everything to chromatic three times, once for windows, once for mac, and once for windows.